ckb-mercury-sdk/src/main/java/org/nervos/mercury/model/common/Record.java

```diff
@@ -15,9 +15,9 @@ public class Record {
   @SerializedName("ownership")
   public Ownership ownership;
 
-  public BigInteger amount;
+  public long amount;
```

Comment:
amount can be udt amount, thus the type is `u128`, so here use `long` as data type is not enough.

Reply:
You are right. Here should be changed